examples/sqs/main.go

```diff
@@ -114,7 +114,7 @@ func (ac *sqsComponent) Process(_ context.Context, btc patronsqs.Batch) {
 		logger := log.FromContext(msg.Context())
 		var u examples.User
 
-		err := json.DecodeRaw(msg.Body(), u)
+		err := json.DecodeRaw(msg.Body(), &u)
```

Comment:
This decoding was failing because it expects a pointer and not a value. This issue was breaking the chain of events.